pom.xml

```diff
@@ -669,12 +692,36 @@
 
     <profiles>
         <profile>
+            <id>non-jigsaw</id>
+            <activation>
+                <!-- The versions we're using of Ehcache, Infinispan and Neo4J won't work on JDK9. Needs upgrading.  -->
+                <jdk>[1.7,1.8]</jdk>
+            </activation>
+            <modules>
+                <module>ehcache</module>
```

Comment:
Se we can't run them at all? Couldn't we exclude single tests? Or is it more global issues with them, causing all tests to fail?